### PR TITLE
Make updateScoreInactiveDocuments correctly update very low scores

### PR DESCRIPTION
**Pull request by Will-Howard**
We were seeing some instances of very old comments not being ordered correctly when sorted by magic. This was because (80% confident this was the only problem) they were being filtered out when updating because the difference between oldScore and newScore wasn't big enough in absolute terms, even though in relative terms it was significant.

I've changed the calculation to use a threshold that is relative to the power of one vote _at the time the comment was posted_, rather than a fixed value.

I've only updated the postgres version.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1204834086579434) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/server/updateScores.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import { getCollection, Vulcan } from "./vulcan-lib";
 import {
-  recalculateScore,
```
Comment (Will-Howard) [inline packages/lesswrong/server/updateScores.ts:3]: Unused import

File: packages/lesswrong/server/updateScores.ts [hunk 2]
```diff
@@ -258,29 +195,34 @@ export const batchUpdateScore = async ({collection, inactive = false, forceUpdat
...
-      updatedDocumentsCounter++;
-      return {
-        updateOne: {
-          filter: {_id: i._id},
-          update: {$set: {score: i.newScore, inactive: false}},
-          upsert: false,
+  const batches = chunk(items, 1000); // divide items into chunks of 1000
+
+  for (let batch of batches) {
+    let itemUpdates = _.compact(batch.map((i: AnyBecauseTodo) => {
```
Comment (jpaddison3) [inline packages/lesswrong/server/updateScores.ts:201]:
```suggestion
    let itemUpdates = compact(batch.map((i: AnyBecauseTodo) => {
```

Comment (jpaddison3) [reply packages/lesswrong/server/updateScores.ts:201]: And import from lodash/compact

**Review by jpaddison3 — APPROVED** [review]
LGTM

**Pull request merged**